library/frr_bgp.py

```diff
@@ -0,0 +1,414 @@
+            description:
+              - Specifies the maximum hop count for EBGP neighbors not on directly connected networks.
+              - The range is from 1 to 255.
+            type: int
+          peer_group:
+            description:
+              - Name of the peer group that the neighbor is a member of.
+          timers:
+            description:
+              - Specifies BGP neighbor timer related configurations.
+            suboptions:
+              keepalive:
+                description:
+                  - Frequency (in seconds) with which the FRR sends keepalive messages to its peer.
+                  - The range is from 0 to 65535.
+                type: int
+                required: True
+              holdtime:
+                description:
+                  - Interval (in seconds) after not receiving a keepalive message that FRR declares a peer dead.
+                  - The range is from 0 to 65535.
+                type: int
+                required: True
+          activate:
+            description:
+              - Enable the address family for this neighbor.
+            type: bool
+          remove_private_as:
+            description:
+              - Remove the private AS number from outbound updates.
+            type: bool
+          next_hop_self:
+            description:
+              - Enable/disable the next hop calculation for this neighbor.
+            type: bool
+          next_hop_unchanged:
+            description:
+              - Enable/disable propagation of next hop unchanged for iBGP paths to this neighbor.
+            type: bool
+      networks:
+        description:
+          - Specify networks to announce via BGP.
+        suboptions:
+          prefix:
+            description:
+              - Network ID to announce via BGP.
+            required: True
+          masklen:
+            description:
+              - Subnet mask length for the network to announce(e.g, 8, 16, 24, etc.).
+          route_map:
+            description:
+              - Route map to modify the attributes.
+      address_family:
+        description:
+          - Specifies BGP address family related configurations.
+        suboptions:
+          afi:
+            description:
+              - Type of address family to configure.
+            choices:
+              - ipv4
+              - ipv6
+            required: True
+          safi:
+            description:
+              - Specifies the type of cast for the address family.
+            choices:
+              - flowspec
+              - unicast
+              - multicast
+              - labeled-unicast
+            default: unicast
+          redistribute:
+            description:
+              - Specifies the redistribute information from another routing protocol.
+            suboptions:
+              protocol:
+                description:
+                  - Specifies the protocol for configuring redistribute information.
```

Comment:
I noticed in one of the code files here that ospfv3 isn't included in the redistribute list.  It makes me think maybe here and in that other PR (#85) we may need to specify the choices allowed? The other PR definitely had ospfv3. (Unless these protocol options are obvious to the users ...)

Reply:
Good catch. I'll add that.